fix build error

🍺 /workspaces/Brewery $ backend
/usr/share/dotnet/sdk/8.0.415/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.DefaultItems.Shared.targets(111,5): warning NETSDK1080: A PackageReference to Microsoft.AspNetCore.App is not necessary when targeting .NET Core 3.0 or higher. If Microsoft.NET.Sdk.Web is used, the shared framework will be referenced automatically. Otherwise, the PackageReference should be replaced with a FrameworkReference. [/workspaces/Brewery/Server/Brewery.Server.Logic/Brewery.Server.Logic.csproj]
/usr/share/dotnet/sdk/8.0.415/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.DefaultItems.Shared.targets(111,5): warning NETSDK1080: A PackageReference to Microsoft.AspNetCore.App is not necessary when targeting .NET Core 3.0 or higher. If Microsoft.NET.Sdk.Web is used, the shared framework will be referenced automatically. Otherwise, the PackageReference should be replaced with a FrameworkReference. [/workspaces/Brewery/Server/Brewery.Server.Logic/Brewery.Server.Logic.csproj]
/workspaces/Brewery/Server/Brewery.ServerMock/BootstrapperMock.cs(5,28): error CS0234: The type or namespace name 'RaspberryPiMock' does not exist in the namespace 'Brewery.Server.Logic' (are you missing an assembly reference?) [/workspaces/Brewery/Server/Brewery.ServerMock/Brewery.ServerMock.csproj]

The build failed. Fix the build errors and run again.

--- a/Server/Brewery.Server.Logic.RaspberryPiMock/Api/GpioModule.cs
+++ b/Server/Brewery.Server.Logic.RaspberryPiMock/Api/GpioModule.cs
@@ -3,7 +3,7 @@ using System.Diagnostics;
 using System;
 using System.Collections.Generic;
 
-namespace Brewery.Server.Logic.RaspberryPi.Api
+namespace Brewery.Server.Logic.RaspberryPiMock.Api
 {
     public class GpioModule : IGpioModule
     {

--- a/Server/Brewery.Server.Logic.RaspberryPiMock/Api/TemperatureModule.cs
+++ b/Server/Brewery.Server.Logic.RaspberryPiMock/Api/TemperatureModule.cs
@@ -1,7 +1,7 @@
 ﻿using Brewery.Server.Core.Api;
 using System.Diagnostics;
 
-namespace Brewery.Server.Logic.RaspberryPi.Api
+namespace Brewery.Server.Logic.RaspberryPiMock.Api
 {
     public class TemperatureModule : ITemperatureModule
     {

--- a/Server/Brewery.Server.Logic/Brewery.Server.Logic.csproj
+++ b/Server/Brewery.Server.Logic/Brewery.Server.Logic.csproj
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
   <ItemGroup>

